easybuild/easyconfigs/g/GPAW/GPAW-19.8.1-foss-2019a-ASE-3.18.0-Python-3.7.2.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = "PythonPackage"
+
+name = 'GPAW'
+version = '19.8.1'
+aseversion = '3.18.0'
+pythonsuffix = '-Python-%(pyver)s'
+versionsuffix = '-ASE-' + aseversion + pythonsuffix
```

Comment:
I don't see the need to include ASE as part of the suffix.

Reply:
No, I incorrectly thought that #8409 would create a version conflict with this one, and that including the ASE version in the suffix would fix it.  But since #8409 has not yet been merged that was silly, the correct solution is to withdraw #8409 for this version.  I will rename the files.